mdlib/db_utils.py

```diff
@@ -46,25 +47,70 @@ def handle_protobuf(self, protobuf_obj):
             case Actions.DELETE_DB:
                 self.delete_db()
 
+    # todo: Consider delete this
+    def __enter__(self):
+        with open(self.db_path, 'r') as db_descriptor:
+            self._tmp_db_dict = json.load(db_descriptor)
+        return self._tmp_db_dict
+
+    # todo: Consider delete this
+    def __exit__(self, exc_type, exc_value, traceback):
+        if traceback:
+            logging.exception(traceback)
+        if exc_type:
+            raise exc_type(exc_value)
+
+        with open(self.db_path, 'w') as db_descriptor:
```

Comment:
This might cause client to dump something to it's local file that the sever does not have